.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -2,7 +2,7 @@
 name: Bug report
 about: Create a report to help us improve pyright
 title: ''
-labels: ''
+labels: bug
```

Comment:
It seems not *all* issues created with this template are manually labelled with `bug` and instead with `enhancement request`. Maybe it should still be the default with the form and manually changed if needed?